CSharp/Microsoft.Bot.Schema.Teams/MessageActionsPayloadBodyExtensions.cs

```diff
@@ -0,0 +1,54 @@
+// <copyright file="MessageActionsPayloadBodyExtensions.cs" company="Microsoft">
+// Licensed under the MIT License.
+// </copyright>
+
+namespace Microsoft.Bot.Schema.Teams
+{
+    using System.Collections.Generic;
+    using HtmlAgilityPack;
+
+    /// <summary>
+    /// MessageActionsPayloadBody extensions.
+    /// </summary>
+
+    public static class MessageActionsPayloadBodyExtensions
+    {
+        /// <summary>
+        /// Strip HTML tags from MessageActionsPayloadBody content.
+        /// </summary>
+        /// <param name="body">The MessageActionsPayloadBody.</param>
+        /// <returns>Plain text content.</returns>
+        public static string GetPlainTextContent(this MessageActionsPayloadBody body)
+        {
+            var doc = new HtmlDocument();
+            doc.LoadHtml(body.content);
+            return StripHtmlTagsHelper(doc.DocumentNode, TextRestrictedHtmlTags);
+        }
+
+        private static readonly HashSet<string> TextRestrictedHtmlTags = new HashSet<string> { "at", "attachment" };
```

Comment:
This should be on top of the class before methods

Reply:
moved